cmd/stack_create_test.go

```diff
@@ -127,21 +137,26 @@ func TestStackCreateInvalidStackCase3(t *testing.T) {
 	sandbox, cleanup := cmdtest.TestSetupWithSandbox(t, true)
 	defer cleanup()
 
-	err := os.RemoveAll("testing-stack")
-	if err != nil {
-		t.Fatal(err)
-	}
+	testStackName := "testing-stack-create-invalid-3"
 
-	args := []string{"stack", "create", "testing-stack", "--copy", "experimental/nodejs"}
-	_, err = cmdtest.RunAppsody(sandbox, args...)
+	args := []string{"stack", "create", testStackName, "--copy", "experimental/nodejs"}
+	_, err := cmdtest.RunAppsody(sandbox, args...)
 
 	if err == nil {
 		t.Fatal(err)
 	}
 
-	exists, err := cmdtest.Exists("testing-stack")
+	exists, err := cmdtest.Exists(testStackName)
+	if err != nil {
+		t.Fatal(err)
+	}
 
 	if exists {
+		// It SHOULDN'T exist, but it might
+		err = os.RemoveAll(testStackName)
+		if err != nil {
+			t.Fatal(err)
+		}
 		t.Fatal(err)
```

Comment:
A better message here would be nice as to why the test is failing. The `err` will be `nil` at this point.